python_scripts/cross_validation_train_test.py

```diff
@@ -168,7 +168,19 @@
 # - train a new model on the train set;
 # - evaluate the testing error on the test set.
```

Comment:
In the paragraph above, can you please link "different cross-validation strategies" to this page of the scikit-learn user guide:

https://scikit-learn.org/stable/modules/cross_validation.html#cross-validation-iterators

Reply:
Done, thanks!